### PR TITLE
Add FAD workflow editor docs

### PR DESCRIPTION
**Pull request by helinanever**
Added a new page about setting up publishing to Firebase App Distribution in Flutter workflow editor

### CODE REVIEW

**Comment by helinanever** [comment]
TODO: add information about release notes

File: content/publishing/firebase-app-distribution.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Set up publishing to [Firebase App Distribution](https://firebase.google.com/docs/app-distribution) to distribute your Android and iOS apps to testers.
+
+{{<notebox>}}
+This guide only applies to workflows configured with the **Flutter workflow editor**. If your workflow is configured with **codemagic.yaml**, please go to [Publishing an app to Firebase App Distribution](../publishing-yaml/distribution/#publishing-an-app-to-firebase-app-distribution).
+{{</notebox>}}
+
+## Requirements
+
+* [Add Firebase to your Flutter project](https://firebase.google.com/docs/flutter/setup?platform=ios)
```
Comment (VeArnold) [inline content/publishing/firebase-app-distribution.md:15]: is linking to ios intentional?

File: content/publishing/firebase-app-distribution.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Generate a [Firebase token](https://firebase.google.com/docs/cli#cli-ci-systems) locally
+* Set up [iOS code signing](../code-signing/ios-code-signing). An Ad Hoc or Enterprise distribution profile is required to distribute the app outside your development team.
```
Comment (VeArnold) [inline content/publishing/firebase-app-distribution.md:17]: development profile works too (ad-hoc and enterprise is only required if you want to publish to people outside of your dev circle)

**Review by VeArnold — CHANGES_REQUESTED** [review]

File: content/publishing/firebase-app-distribution.md (hunk 2 above)
Comment (VeArnold) [reply content/publishing/firebase-app-distribution.md:17]: maybe good to add this back in somewhere too (like on android)

{{<notebox>}} In addition, each uploaded binary must have a different version to appear in Firebase console, see how to automatically [increment build version](../building/build-versioning/ 'Build versioning') on Codemagic. {{</notebox>}}

File: content/publishing/firebase-app-distribution.md (hunk 1 above)
Comment (helinanever) [reply content/publishing/firebase-app-distribution.md:15]: no, but the page seems to take iOS as the argument by default, I'll update

File: content/publishing/firebase-app-distribution.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* Set up [Android code signing](../code-signing/android-code-signing). If you do not set up code signing, the artifact will be signed with a debug keystore from Codemagic build machine.
+
+## Enabling publishing to Firebase App Distribution
+
+1. Navigate to **App settings > Distribution > Firebase App Distribution**.
+2. Enter your **Firebase token**.
+3. Provide the **Firebase app ID** for Android and/or iOS. Note that the fields for Android or iOS configuration are displayed conditionally based on the selected build platforms at the top of the page.
+4. Enter the name(s) of the **tester groups** to whom you want to distribute your Android or iOS app. The field is case-sensitive. To enter multiple groups, separate them with a comma.
+5. Select **Enable publishing to Firebase App Distribution** at the top of the section to enable publishing.
```
[the remark this answers is not in the document]
Comment (helinanever) [reply content/publishing/firebase-app-distribution.md:26]: good point

**Review by VeArnold — APPROVED** [review]

**Review by VladyslavBondarenko — APPROVED** [review]

**Pull request merged**